code/stable/glassbr/SRS/HTML/GlassBR_SRS.html

```diff
@@ -552,7 +552,7 @@ <h2>Abbreviations and Acronyms</h2>
       <div class="section">
         <h1>Introduction</h1>
         <p class="paragraph">
-          Software is helpful to efficiently and correctly predict the blast risk involved with the glass slab. The blast under consideration is any kind of man-made explosion. The software, herein called GlassBR, aims to predict the blast risk involved with the glass slab using an intuitive interface.
+          Explosion in downtown areas are dangerous from the blast itself and also potentially from the secondary effect of falling glass. Therefore, software is needed to predict whether a glass slab can withstand a blast under given conditions. For example, we might wish to know whether a pane of glass from a gas main explosion from a small fertilizer truck bomb. The program documented here is called GlassBR.
```

Comment:
Should be "Explosion**s**"; not sure what variant of `atStart` would do this (if any), so you might need to look into `plural` as well.

Also should be "…whether a pane of glass **fails** from…"

Is the gas main explosion caused by the small fertilizer truck bomb here, or are they two separate examples (and thus should be separated by "or")? @smiths

Reply:
It does seem to make it uppercase - maybe `explosion` is done using a phrase constructor that prevents pluralization?

Reply:
@NoahCardoso and I looked at it together just now. It looks like the majority of the "words" defined in GlassBR's `Unitals.hs` use `nounPhraseSP` (i.e., "noun phrase that is self-plural"). Our first idea was to use `nounPhrase` to manually add the `s` ourselves, but that doesn't seem to be too commonly used, and when it is used, it's used for a complex phrase. The same issue occurs with other examples, such as Projectile and PD Controller. It's not quite clear how we should proceed -- is there a 'right' way to go about this? Is `nounPhrase` just what we want to use then?

Reply:
RE: the local README.md: it looks to be largely out of date and duplicates information that should exist in the Haddock docs. Shall we remove the out of date information, without replacement?

Reply:
Ah, it looks like definition of `explosion` should use `cn'` to make an `NP`. A smidge bit hidden.